src/select-list-view.ts

```diff
@@ -413,8 +440,15 @@ module.exports = class SelectListView {
   }
 }
 
+type ListItemViewProps = { element: EtchElement ; selected: boolean; onclick: () => void }
```

Comment:
Doesn't need to be changed, but just wondering why you chose to use `type` here instead of an interface (though that may have been because I mentioned "type" in my comment 😁).